### ISSUE TITLE
Show Unlocked Tab Titles in Italics like VSCode

### ISSUE THREAD

**Issue by ei-eiron** [tabwrangler/tabwrangler#419]
**Is your feature request related to a problem? Please describe.**
I only found this extension by searching for a similar feature to preview tabs in vscode.. but..
I have to click on the tab to see which tabs I've locked, and which are unlocked.

**Describe the solution you'd like**
One or the other should have some unobtrusive visual indicator, like:
- Page title in italics (like VSCode Preview mode vs Fully Opened)

**Describe alternatives you've considered**
Pinning the tab. Not good, since I can't see the title, and pinning has other implications.

**Comment by ssorallen** [comment]
I do not believe this is possible without a change from browser developers, and it would minimally require an additional permission from Tab Wrangler (injecting CSS into any host via [`host_permissions`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions)). This is a nice request but unlikely to be possible any time in the near future.

**Comment by ei-eiron** [comment]
Thanks @ssorallen for taking a look. Apologies for late response. The extension is significantly improving my workflow and memory usage. 

I don't know much about what's involved in decorating tabs.. 😓 

I was thinking there might be extensions out there that use alert indicators of some kind like the green "updated" dot or the "playing sound" icon. Admittedly I haven't even googled this, and I can see how Firefox would want tight control over perms for that.
